hls-metadata-0.0.1.json

```diff
@@ -1431,5 +1431,79 @@
                 "9.8.1"
             ]
         }
+    },
+    "2.5.0.0": {
```

Comment:
We have to drop `hls-metadata`, because it corresponds to the default release channel, not vanilla (it's used by the vscode-haskell extension).

Reply:
We switch the vscode extenstion to using the vanilla channel in https://github.com/haskell/vscode-haskell/pull/999

Reply:
I don't think we should.